tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -116,7 +116,7 @@ def test_stack_root_mismatch_error(self):
         response = self.get_success_response(
             self.organization.slug, self.project.slug, qs_params={"file": "wrong/file/path"}
         )
-        assert response.data["config"] == self.expected_configurations()
+        assert response.data["config"] is None
```

Comment:
This is the only test that matched no code mappings, thus, no need to include it as part of the response.